### DISCUSSION TITLE
Graphql kotlin in SpringBoot 3.X.X

### DISCUSSION THREAD

**Discussion by KjellBerlin** [ExpediaGroup/graphql-kotlin#1784] [Q&A] [answered]
Hello,

when trying to use this library within a SpringBoot project with version 3.X.X it is not possible to access the playground. Is it possible to use this library in a SpringBoot project with a 3.X.X version already? If not is there any plan to enable this soon?

I really like this library and would love to use it further :) 

Thanks in advance!

**Comment by dariuszkuc** [comment, marked as answer]
Hello,
[GraphQL Playground](https://github.com/graphql/graphql-playground) is not actively maintained (and haven't been in a while). With GraphQL Kotlin v7 we now enable [GraphiQL](https://github.com/graphql/graphiql) (an official GraphQL IDE) by default (property `graphql.graphiql.enabled=true`) . You can still opt-in to using playground by specifying `graphql.playground.enabled=true` in your configuration properties.

Thanks,
Derek

**Reply by KjellBerlin** [reply]
Thanks a lot for the fast reply!

**Comment by samuelAndalon** [comment]
Just to add to what @dariuszkuc mentioned, `graphql-kotlin-spring-server` v7 will come with spring boot 3, you can try one of the multiple alpha releases we have done for v7 -- https://github.com/ExpediaGroup/graphql-kotlin/releases/tag/7.0.0-alpha.5

**Reply by chriskn** [reply]
Hi @samuelAndalon, 

can you estimate when a new release of v7 will be available? We are currently stuck with Spring-Boot 2.7 because graphql-kotlin and the broken federated type resolver in 7.0.0-alpha.5. It would help us if you could release a new alpha, including https://github.com/ExpediaGroup/graphql-kotlin/pull/1753. 

By the way: Version 6.5.0 has also Spring-Boot related issues. Spring-Boot-Dependencies 2.7.x uses graphql-java 18.5 and the newly used `LightDataFetcher` is not available in graphql-java 18.5. This leads to a class not found exception if you don't force version 20.3 of graphql-java. 

Thanks in advance!

**Reply by samuelAndalon** [reply]
Just released [7.0.0-alpha.6](https://github.com/ExpediaGroup/graphql-kotlin/releases/tag/7.0.0-alpha.6),

for the 6.5.0 issue, you can always override the graphql-java version that springboot bom pulls, that way you can use the `LightDataFetcher `
